### ISSUE TITLE
Termos similares apontando para o nome exato

### ISSUE THREAD

**Issue by stees** [geoinfo-smdu/geosampa-mais-livre#6]
Seria interessante "ruas" apontar pra "logradouros", "árvore" apontar pra arborização viária, "rios" e "córregos" para "drenagem", pra dar apenas alguns exemplos.

Imagino que seriam essas as palavras que as pessoas pensariam pra procurar, em vez das oficiais.

**Comment by feromes** [comment]
Verdade! Atualmente a busca procura no nome da camada apenas, mas estamos planejando um sistema mais complexo que pondere palavras nos Metadados.
